### DISCUSSION TITLE
next-server virtual memory

### DISCUSSION THREAD

**Discussion by quicksandznzn** [vercel/next.js#66067] [Help]
### Summary
Next.js Version 14.1.0

Why next-server need 20g virtual memory

<img width="1044" alt="image" src="https://github.com/vercel/next.js/assets/23514918/2b003037-da4f-49b7-95ba-754c26ee8cf1">

**Dockerfile**
`FROM node:20.11-alpine3.19`

**Comment by drm-jinsuo1o7** [comment]
Same problem. Is there any solution?

I ran nextjs 14.1.1 as a docker container. The same problem occurred and I am reserving 20.7g of virtual memory. 
The actual memory used is small, but why does this come up when I check it with the htop command?

**Reply by Enkratia** [reply]
Hello, did you find answer?

**Reply by drm-jinsuo1o7** [reply]
No, I couldn't find a solution.
